src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.zh-Hans.xlf

```diff
@@ -0,0 +1,602 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" source-language="en" target-language="zh-CN" original="../OpenIddictResources.resx">
```

Comment:
Please replace `zh-CN` to `zh-Hans` to match the file name.

Reply:
uh, I missed it...